<launch>
  <!-- Machine definition -->
  <machine name="boat_fs" address="192.168.10.4" user="reeve" password="T@nner38565796h"  timeout="5000" env-loader="/home/reeve/catkin_ws/devel/env.sh"/>
  <machine name="boat_bs" address="192.168.10.5" user="pi"    password="spring2020"       timeout="5000" env-loader="/home/pi/catkin_ws/devel/env.sh"/>
  <!-- Nodes to be executed in backseat -->
  <node machine="boat_bs" name="XcvrAnalysis"       pkg="seatrac_pkg"     type="AcousticAnalysis.py" output="screen"/>
  <!-- Uncomment designated Path Planning Node to Launch for the Boat -->
  <!--node machine="boat_bs" name="pp_manual" pkg="pm_plan" type="pp_manual.py"/-->

  <!-- Nodes to be executed in frontseat -->
  <node machine="boat_fs" name="PWMcntrl_left"      pkg="aut_cat"         type="PWMcntrl_left.py" />
  <node machine="boat_fs" name="PWMcntrl_right"     pkg="aut_cat"         type="PWMcntrl_right.py" />
  <node machine="boat_fs" name="GPS_Calculations_2" pkg="aut_cat"         type="GPS_Calculations_2.py" />
  <node machine="boat_fs" name="GPS_Driver"         pkg="aut_cat"         type="GPS_Driver.py" />
  <node machine="boat_fs" name="logtoCSV"           pkg="aut_cat"         type="iverLog.py" />
  <node machine="boat_fs" name="PID_rot"            pkg="aut_cat"         type="PID_rot.py" />
  <node machine="boat_fs" name="PID_vel"            pkg="aut_cat"         type="PID_vel.py" />
  <node machine="boat_fs" name="Motor_Control"      pkg="aut_cat"         type="Motor_Control.py" />
  <node machine="boat_fs" name="Main"               pkg="aut_cat"         type="Main.py" />
  <node machine="boat_fs" name="Imuuu"              pkg="aut_cat"         type="Imuuu.py"/>
  <node machine="boat_fs" name="Radio"              pkg="aut_cat"         type="radio.py"/>
</launch>
